tests/ut/xrd-control-plane/cp_configmap.bats

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env bats
```

Comment:
Note cp/vr prefixes are now redundant - you can remove the prefixes from all the .bats files.